vertica/datadog_checks/vertica/vertica.py

```diff
@@ -44,7 +50,7 @@ def __init__(self, name, init_config, instances):
         self._timeout = float(self.instance.get('timeout', 10))
         self._tags = self.instance.get('tags', [])
 
-        self._client_lib_log_level = self.instance.get('client_lib_log_level')
+        self._client_lib_log_level = self.instance.get('client_lib_log_level', datadog_agent.get_config('log_level'))
```

Comment:
```suggestion
        self._client_lib_log_level = self.instance.get('client_lib_log_level', logging.getLevelName(self.log.getEffectiveLevel()))
```

logging from `import logging`

@florimondmanca Maybe ☝️  this is more reliable ?

Reply:
Good call, looks more robust.